### DISCUSSION TITLE
miactl authentication - Invalid redirect URI during login phase

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#359] [Q&A] [answered]
### Which section?

Console

### Which is the version?

miactl 0.9.0

### Description

Hi guys,
I tried to login using the `miactl` on a On Premise instance of the console.
The login is performed through GitLab but, after logging in, I got the following error:
`The redirect URI included is not valid.`

Do I need to add `localhost` as a redirect URI inside the GitLab Application configuration?

**Comment by JGiola** [comment]
Hello, there is a gap in the documentation for miactl for setting up properly the authorization provider. I'm updating it and will post here the link.

**Reply by JGiola** [reply, marked as answer]
Here the new paragraph about the needed configuration

https://github.com/mia-platform/miactl/blob/main/docs/20_setup.md#setup-authorization-provider

It will be added to the public documentation site ASAP

**Reply by frengooo** [reply]
Thanks @JGiola for your reply 😄 I'll try
